packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -584,16 +593,16 @@ export class GarbageCollector implements IGarbageCollector {
          * Whether sweep should run or not. The following conditions have to be met to run sweep:
          *
          * 1. Overall GC or mark phase must be enabled (this.shouldRunGC).
-         *
          * 2. Sweep timeout should be available. Without this, we wouldn't know when an object should be deleted.
-         *
-         * 3. Sweep should be enabled for this container (this.sweepEnabled). This can be overridden via runSweep
+         * 3. The driver must implement the policy limiting the age of snapshots used for loading. Otherwise
+         * the Sweep Timeout calculation is not valid. We use the persisted value to ensure consistency over time.
+         * 4. Sweep should be enabled for this container (this.sweepEnabled). This can be overridden via runSweep
          * feature flag.
          */
-        this.shouldRunSweep = false; // disable while TEMPORARY measure hardcoding snapshotCacheExpiryMs is here
-            // this.shouldRunGC
-            // && this.sweepTimeoutMs !== undefined
-            // && (this.mc.config.getBoolean(runSweepKey) ?? this.sweepEnabled);
+        this.shouldRunSweep =
+            this.shouldRunGC
+            && this.sweepTimeoutMs !== undefined
+            && (this.mc.config.getBoolean(runSweepKey) ?? this.sweepEnabled);
```

Comment:
`sweepEnabled` -> `sweepForceEnabled`?

Reply:
Can you say more why you're suggeting that?  I'd more say the config setting is the "forceful" one - `this.sweepEnabled` is the value computed on create and written to the file in the initial snapshot.

Reply:
Specifically `testOverrideSweepTimeoutMs !== undefined` is in the condition of `sweepEnabled`, maybe I'd rather put that in `this.shouldRunSweep`

Reply:
It has to go here so the values persisted to the file are coherent.  shouldRunSweep is about this session, not the file's capabilities. I'm inclined to leave it as-is.